authcontroller.py

```diff
@@ -490,8 +495,9 @@ def invite_organisation():
                         "email/org_invitation.html",
```

Comment:
Perhaps we can also embed the organisation name in the token, so that we can prefill the confirmation form using also the name.